preload_data.sh

```diff
@@ -138,6 +138,20 @@ function prereq() {
         error "Namespace $TO_NAMESPACE does not exist (or oc command line is not logged in)"
         exit 1
     fi
+    mongo_node=$(${OC} get pods -n $FROM_NAMESPACE -o wide | grep icp-mongodb-0 | awk '{print $7}')
+    architecture=$(${OC} describe node $mongo_node | grep "Architecture:" | awk '{print $2}')
+    if [[ $architecture == "s390x" ]]; then
+      s390x_ENV="true"
+      info "Z or Power cluster detected, be prepared for multiple restarts of mongo pods. This is expected behavior."
```

Comment:
@bluzarraga 

May I ask that if Z and Power Cluster both require non-ssl connection, should we update the architecture variable to `s390x` or `ppc64le`?

Reply:
Yes, good catch